src/api/utils/summary_cache.py

```diff
@@ -202,9 +203,21 @@ def _update_project_summary_sample_events(
 
         proj_summary_se_model.objects.filter(project_id=project_id).delete()
         tags = [{"id": str(t.pk), "name": t.name} for t in project.tags.all()]
+        admins = project.profiles.filter(role=ProjectProfile.ADMIN)
+        project_admins = [{"id": str(pa.pk), "name": pa.profile_name} for pa in admins]
+        data_policies = dict(Project.DATA_POLICIES)
+
         proj_summary_se_model.objects.create(
             project_id=project_id,
             project_name=project.name,
+            project_admins=project_admins,
+            project_notes=project.notes,
+            data_policy_beltfish=data_policies.get(project.data_policy_beltfish),
```

Comment:
if they don't exist should they either default to strict or what we currently have set as default?